src/main/java/com/dsb/gui/MainController.java

```diff
@@ -51,7 +51,7 @@ protected void onUpdateButtonClick() {
         } catch (Exception e) {
             // TODO Notify user about error in GUI
             System.out.println("Blad mordo");
-            e.printStackTrace();
+            System.out.println("Something not right with mainController, xDdddd...");
```

Comment:
Przywróć na razie wszędzie printStackTrace(), błędy zsuppressuj albo wgl będziemy ignorować, potem się jakoś inaczej to ogarnie ale na razie musimy wiedzieć co dokładnie tam leci w razie co